### DISCUSSION TITLE
Some tips for optimization

### DISCUSSION THREAD

**Discussion by MissNanLan** [lencx/Noi#117] [General]
**1、 tag多选框遮住了ok按钮， 不然每次选完tag后，还得关闭多选框，才能去点击ok按钮**

<img width="613" alt="image" src="https://github.com/lencx/Noi/assets/22790863/d79f0c82-ec0c-4f65-a294-8f93a58507fd">

**2、侧边栏能否增加一个编辑按钮，直接去Promopt设置界面，现在只有一个拷贝。因为每次要添加或者修改，都得从设置里面打开**

<img width="1190" alt="image" src="https://github.com/lencx/Noi/assets/22790863/4c7bc39e-d53f-4caf-aff7-6dea8c99ba08">

**Comment by lencx** [comment]
1. 选完 Tag 按 ESC 键也可以退出，这个暂时也没想到比较好的交互。
2. Prompts 新增修改其实可以直接通过弹窗编辑来解决，只不过需要设计一下交互的细节。

**Reply by MissNanLan** [reply]
1、 ESC是退出，我是要确认
